tools/filterwriter/filterwriter.go

```diff
@@ -20,17 +20,19 @@ type Writer struct {
 	writer  io.Writer
 	secrets [][][]byte
 
-	chunk []byte
-	store [][]byte
-	mux   sync.Mutex
+	chunk     []byte
+	store     [][]byte
+	mux       sync.Mutex
+	lastTimer *time.Timer
 }
 
 // New ...
 func New(secrets []string, target io.Writer) *Writer {
-	return &Writer{
+	writer := &Writer{
```

Comment:
what is the reason for this change? it leads to an extra line, but i do not see any advantage, am i missing something?